src/main/java/com/facebook/openwifirrm/ChannelWidth.java

```diff
@@ -0,0 +1,8 @@
+package com.facebook.openwifirrm;
+
+/**
+ * Represents the possible channel widths allowed in the 5 GHz band.
+ */
+public enum ChannelWidth {
+	MHz_20, MHz_40, MHz_80, MHz_160, MHz_80_PLUS_80
+}
```

Comment:
I am not convinced this class is useful, and also the naming is bad (if this is specifically for 5GHz widths, that should be in the enum name)

Reply:
It is not meant specifically for 5GHz widths. But in any case, I can remove it, since it is used in the operation element classes for encoding which you asked me to remove.